config/pmd/pmd.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<ruleset name="Custom Rules">
+    <description>
+        PMD configuration
+    </description>
+
+    <rule ref="category/java/bestpractices.xml">
+        <exclude name="MethodReturnsInternalArray"/>
+    </rule>
+
+    <rule ref="category/java/codestyle.xml">
+        <exclude name="CallSuperInConstructor"/>
```

Comment:
Why this rule has beed excluded?

Reply:
This rule has conflict with checkstyles configuration, one of them need to be excluded

Reply:
Alright